CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.6)
+project(cpp_study)
+set(CMAKE_CXX_STANDARD 14)
+
+include_directories("${PROJECT_BINARY_DIR}")
+set(SOURCE_FILES  tutorial.cpp tutorial.h)
+
+add_executable(cpp_study ${SOURCE_FILES} tutorial.cpp tutorial.h)
```

Comment:
IMO, there is no need to add a dedicate cpp_study executable here, as you will see there are several dedicate examples  with  main entry function in section2, section3 ...
Maybe a good practice is add_executable under each sub directory and just include_directories for all section* in the top-level CMakeLists.txt.